core/metacling/src/TCling.cxx

```diff
@@ -1369,17 +1369,12 @@ TCling::TCling(const char *name, const char *title, const char* const argv[])
    if (fromRootCling) {
       fInterpreter->declare("#include \"RtypesCore.h\"\n"
                             "#include <string>\n"
-                            "using std::string;\n"
-                            "#include <cassert>\n");
```

Comment:
This changes may or may not good/appropriate **but** definitively do not fall under the commit's description.   Can you put them in a separate commit with an explanation ...  thanks.